### PR TITLE
Allow selection of message counts to display

### PR DESCRIPTION
**Pull request by Greybird**
The purpose of this PR is to allow configuring the message counts that the user wants to display.

It defaults to the current display (Active/DeadLetter/Transfer DeadLetter), but also allows to select Scheduled/Transfer message counts as well. Order is not modifiable but this should already bring some value to users.

> Note that this PR might conflict with #468 due to the modification of the same code zones. I will happily handle the merge once any will be merged into the repo

# Screenshots
## Configuration
image
## Results
Sorry, all counts are at 0, but you can get the idea
image

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-counts.71.build.714 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/35560044) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/8903d66b94 by @)

File: src/Common/Helpers/Constants.cs [hunk 1]
```diff
@@ -37,7 +37,12 @@ public static class Constants
         public const string EventHubEntities = "Event Hubs";
         public const string NotificationHubEntities = "Notification Hubs";
 
-
+        public const string ActiveMessages = "Active";
+        public const string DeadLetterMessages = "DeadLetter";
```
Comment (SeanFeldman) [inline src/Common/Helpers/Constants.cs:41]:
```suggestion
        public const string DeadLetterMessages = "DeadLettered";
```

File: src/Common/Helpers/Constants.cs [hunk 2]
```diff
@@ -37,7 +37,12 @@ public static class Constants
...
+        public const string ScheduledMessages = "Scheduled";
+        public const string TransferMessages = "Transfer";
```
Comment (SeanFeldman) [inline src/Common/Helpers/Constants.cs:43]: These are messages being transfered. I wonder if the name should reflect that.

```suggestion
        public const string TransferMessages = "InTransfer";
```

File: src/ServiceBusExplorer.sln.DotSettings [hunk 3]
```diff
@@ -590,6 +590,9 @@ II.2.12 &lt;HandlesEvent /&gt;&#xD;
...
 	<s:String x:Key="/Default/CodeStyle/Naming/XamlNaming/UserRules/=XAML_005FRESOURCE/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpFileLayoutPatternsUpgrade/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpKeepExistingMigration/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpPlaceEmbeddedOnSameLineMigration/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpUseContinuousIndentInsideBracesMigration/@EntryIndexedValue">True</s:Boolean>
```
Comment (SeanFeldman) [inline src/ServiceBusExplorer.sln.DotSettings:595]: Likely unintended change.
```suggestion
```

**Review by SeanFeldman — CHANGES_REQUESTED** [review]

File: src/Common/Helpers/Constants.cs (hunk 2 above)
Comment (Greybird) [reply src/Common/Helpers/Constants.cs]: Seems like a good idea to me.

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-counts.72.build.715 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/35565954) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/7e06ccc7f1 by @Greybird)

File: src/ServiceBusExplorer.sln.DotSettings (hunk 3 above)
Comment (Greybird) [reply src/ServiceBusExplorer.sln.DotSettings]: Sorry, missed that at commit & PR time 😞

**Review by SeanFeldman — APPROVED** [review]

**Review by ErikMogensen — APPROVED** [review]
Very good!

**Comment by paolosalvatori** [comment]
This is AWESOME, thanks @Greybird :)

**Pull request merged**